### ISSUE TITLE
unable to upload new file with rotation EXIF

### ISSUE THREAD

**Issue by theonlybenjamin** [Donaldcwl/browser-image-compression#148]
hello, I want to upload the new file with EXIF ​​​​rotation to my server, it does not show me any error in the console, but my file is uploaded without EXIF ​​rotation. here's my code
![image](https://user-images.githubusercontent.com/47691286/166174950-206998cc-d778-4803-98e5-8ecbce1ddde5.png)

**Comment by Donaldcwl** [comment]
This lib reads the EXIF rotation information and rotates it accordingly. It is normal that the output file does not have EXIF information.

**Comment by theonlybenjamin** [comment]
Yes, want i want to do is upload the rotated file, and dont need the exif information on the file, i just need the rotated file but it(the lib) doesnt rotate the file.
The exif rotation number is 6 but my image arrives like this to the server(original image, withouth rotation)
![image](https://user-images.githubusercontent.com/47691286/166390207-0d0ddb35-070a-428d-860f-49a0b6c069e2.png)

**Comment by theonlybenjamin** [comment]
maybe im doing something wrong, please I would really appreciate your help

**Comment by Donaldcwl** [comment]
@theonlybenjamin It seems that your image file is png, you may provide the `exifOrientation` value in the option. For example:
```javascript
options = {
  maxSizeMB: 1,
  useWebWorker: true,
  exifOrientation: 6
}
```

We only detect EXIF rotation information for jpg images at the moment.

**Comment by theonlybenjamin** [comment]
its jpg image, give you some pictures:
this is the file:
![image](https://user-images.githubusercontent.com/47691286/166588125-3571de31-537a-40cb-9429-5ea7d2e269d6.png)
uploading the file:
![image](https://user-images.githubusercontent.com/47691286/166588311-d4b10cd6-6fd4-42b7-8208-0d98f4fc90ca.png)
and this is the code:
![image](https://user-images.githubusercontent.com/47691286/166588384-a714c5df-9465-41ef-8cc0-e3407d2836e3.png)
Even if i harcoded the exifOrientation property with 6, it doesnt rotate it

**Comment by Donaldcwl** [comment]
Can you zip the jpg file and upload it here?

> its jpg image, give you some pictures: this is the file: ![image](https://user-images.githubusercontent.com/47691286/166588125-3571de31-537a-40cb-9429-5ea7d2e269d6.png) uploading the file: ![image](https://user-images.githubusercontent.com/47691286/166588311-d4b10cd6-6fd4-42b7-8208-0d98f4fc90ca.png) and this is the code: ![image](https://user-images.githubusercontent.com/47691286/166588384-a714c5df-9465-41ef-8cc0-e3407d2836e3.png) Even if i harcoded the exifOrientation property with 6, it doesnt rotate it

**Comment by theonlybenjamin** [comment]
of course, there you go
[test.zip](https://github.com/Donaldcwl/browser-image-compression/files/8648012/test.zip)

**Comment by Donaldcwl** [comment]
![image](https://user-images.githubusercontent.com/11781337/172038035-61450a4c-59af-4c09-8e49-aa6523a9b86e.png)
[test_result.zip](https://github.com/Donaldcwl/browser-image-compression/files/8839426/test_result.zip)
@theonlybenjamin I cannot reproduce your issue, the lib detects it's exifOrientation value is 6 and rotated it accordingly.
You may try it out in https://donaldcwl.github.io/browser-image-compression/example/basic.html
I have upload my result here for your reference.
